chart/load-balancer-api/values.yaml

```diff
@@ -1,5 +1,5 @@
 image:
-  repository: ghcr.io/infratographer/load-balancer-api
+  repository: ghcr.io/infratographr/load-balancer-api
```

Comment:
typo still? The original `infratographer/load-balancer-api` image works for me

Reply:
typing is hard.... 😆